datarepo-clienttests/src/main/java/testscripts/RetrieveSnapshot.java

```diff
@@ -0,0 +1,179 @@
+package testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.api.ResourcesApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.BillingProfileModel;
+import bio.terra.datarepo.model.BulkLoadArrayRequestModel;
+import bio.terra.datarepo.model.BulkLoadArrayResultModel;
+import bio.terra.datarepo.model.BulkLoadFileModel;
+import bio.terra.datarepo.model.DatasetSummaryModel;
+import bio.terra.datarepo.model.DeleteResponseModel;
+import bio.terra.datarepo.model.IngestRequestModel;
+import bio.terra.datarepo.model.IngestResponseModel;
+import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.SnapshotModel;
+import bio.terra.datarepo.model.SnapshotSummaryModel;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import utils.DataRepoUtils;
+import utils.FileUtils;
+
+public class RetrieveSnapshot extends runner.TestScript {
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public RetrieveSnapshot() {
+    super();
+  }
+
+  private String datasetCreator;
+  private BillingProfileModel billingProfileModel;
+  private DatasetSummaryModel datasetSummaryModel;
+  private SnapshotSummaryModel snapshotSummaryModel;
+
+  private Storage storage;
+  private String testConfigGetIngestbucket;
+
+  public void setup(Map<String, ApiClient> apiClients) throws Exception {
+    // pick the first user to be the dataset creator
+    List<String> apiClientList = new ArrayList<>(apiClients.keySet());
+    datasetCreator = apiClientList.get(0);
+
+    // get the ApiClient for the dataset creator --- and the snapshot creator?
+    ApiClient datasetCreatorClient = apiClients.get(datasetCreator);
+    ResourcesApi resourcesApi = new ResourcesApi(datasetCreatorClient);
+    RepositoryApi repositoryApi = new RepositoryApi(datasetCreatorClient);
+
+    storage = StorageOptions.getDefaultInstance().getService();
+    testConfigGetIngestbucket = "jade-testdata"; // this could be put in DRUtils
+
+    // create a new profile
+    billingProfileModel =
+        DataRepoUtils.createProfile(resourcesApi, billingAccount, "profile-simple", true);
+    System.out.println("successfully created profile: " + billingProfileModel.getProfileName());
```

Comment:
Could you change all the System.out.println to logger.info or logger.debug? I think you can use info level in setup/cleanup methods and debug level in userJourney method.